_delphi_utils_python/delphi_utils/validator/report.py

```diff
@@ -74,21 +78,25 @@ def add_raised_warning(self, warning):
         """
         self.raised_warnings.append(warning)
 
-    def set_summary(self):
-        """Represent summary of report as a string."""
-        out_str = f"{self.total_checks} checks run\n"
-        out_str += f"{len(self.unsuppressed_errors)} checks failed\n"
-        out_str += f"{self.num_suppressed} checks suppressed\n"
-        out_str += f"{len(self.raised_warnings)} warnings\n"
-        self.summary = out_str
-
     def log(self, logger=None):
         """Log errors and warnings."""
         if logger is None:
             logger = get_structured_logger(__name__)
 
-        self.set_summary()
-        logger.info(self.summary)
+        if self.success():
+            logger.info("Validation run successful",
+                data_source = self.data_source,
+                checks_run = self.total_checks,
+                checks_failed = len(self.unsuppressed_errors),
+                checks_suppressed = self.num_suppressed,
+                warnings = len(self.raised_warnings))
```

Comment:
So each of these turns into a seperate field in the logs?

Reply:
each key-value pair is a separate column, yes